DuckDuckGo/SettingsViewModel.swift

```diff
@@ -390,6 +393,15 @@ extension SettingsViewModel {
                 self?.state.subscription.canPurchase = !products.isEmpty
             }.store(in: &cancellables)
     }
+        
+    private func setupNotificationObservers() {
+        signOutObserver = NotificationCenter.default.addObserver(forName: .accountDidSignOut, object: nil, queue: .main) { [weak self] _ in
+            if #available(iOS 15.0, *) {
+                self?.signOutUser()
```

Comment:
Doesn't it causes a loop?
`signOutUser()` -> `AccountManager().signOut()` -> posts `.accountDidSignOut`